addons/survey/report/survey_templates.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <template id="certification_report_view_general">
+            <!-- Style classes to be applyed to '#o_survey_certification': [no class](purple), gold, blue -->
+            <div id="o_survey_certification" t-att-data-oe-model="user_input._name" t-att-data-oe-id="user_input.id" t-att-class="'article certification-wrapper %s %s' % (layout_color,layout_template)">
+                <div class="certification">
+                    <div class="certification-seal" t-if="user_input.scoring_success and layout_template == 'modern'"/>
```

Comment:
Maybe it was already discussed, but wouldn't it also reduce the complexity (in addition to helping extensibility) if we indeed introduced the `general` template but kept `_modern` and `_classic` as extensions?

Reply:
Hey, thanks for checking the pr! Do you mean having _modern and _classic views still in the report separately from general template? 
If that's the case, I think it would add even more complexity no?  If user wants to make general changes in the template, they will have to change all three templates separately, and same goes for us, we would have to change code in 3 different places every time something changes. 
I think it should be better to have 2-3 t-if statements in the layout and save ourselves all the trouble (and trim a code a bit), but if something we can discuss it together with Frgi :)

Reply:
We figured out that there would be no point in having both templates in the end if I remember correctly.